### PR TITLE
Reservation skeleton

### PR DESCRIPTION
**Pull request by LittleJennie**
Reservation front end skeleton with CSS.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,22 +1,25 @@
 const path = require('path');
 
 module.exports = {
-  entry: path.resolve(__dirname, 'client', 'index.js'),
+  entry: path.resolve(__dirname, 'client', 'index.jsx'),
```
Comment (cminho921) [inline webpack.config.js:4]: why did you change your extension? Just curious what bugs were here before you change the extension to jsx.

**Review by cminho921 — COMMENTED** [review]
Overall, It's perfect!

File: webpack.config.js (hunk 1 above)
Comment (LittleJennie) [reply webpack.config.js:4]: The index in Client directory supposed to be .jsx in the first place. eslint is giving me an error when I import .jsx files to a .js file.
